src/pages/account/logins/index.tsx

```diff
@@ -62,6 +62,7 @@ const Logins: React.FC<LoginsProps> = ({ sortedIdentities }) => {
       <br />
       <IdentitySection>
         <h3>Email addresses</h3>
+        <br />
```

Comment:
You should remove all the `<br />` and instead do:

```css
...
h3 {
  margin-bottom: ...
}
```

Reply:
OK fixed 16506b1